src/js/jsx/shared/Datalist.jsx

```diff
@@ -314,7 +316,8 @@ define(function (require, exports, module) {
                 case "Tab":
                     return;
                 case "Enter":
-                case "Return":
```

Comment:
Also, the stock (or whatever default) result should work just like every other result. That means I should be able to click on it. But, I can't click on the stock result.